presto-docs/src/main/sphinx/release/release-0.235.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.235
+=============
+
+General Changes
+_______________
+* Fix named queries where its output does not match its column list (:issue:`14333`).
+* Fix error in analysis phase.
+* Fix ZSTD compression issue with zero row file for missing buckets.
+* Improve performance of ``UNNEST``.
+* Improve ``CREATE FUNCTION`` to allow parameter type list and return type to have a length up to 30k characters.
+* Add the ability to pre-check method bytecode size is within limits.
+* Add support for ``SHOW CREATE FUNCTION``.
+* Add optimization for page projection and filter by extract and compute common subexpressions among all projections first. This optimization can be turned off by session property ``optimize_common_sub_expressions``.
```

Comment:
This note is a bit confusing to me. Would the following be correct?
```
Add optimization for filters and projections to extract common subexpressions 
and evaluate them only once.  This optimization can be turned off by the 
session property ``optimize_common_sub_expressions``.
```

Reply:
Is there a Github issue I could reference that someone could click into to understand that?  I'm not sure it's worth it or necessary to add the additional distinction you mention above in the release notes, but it would make a good read in a Github issue.